src/Microsoft.Diagnostics.Monitoring.Options/GlobalCounterOptions.cs

```diff
@@ -32,6 +33,11 @@ public class GlobalCounterOptions
         [DefaultValue(GlobalCounterOptionsDefaults.MaxTimeSeries)]
         [Range(1, int.MaxValue)]
         public int? MaxTimeSeries { get; set; }
+
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_GlobalCounterOptions_ProviderIntervals))]
+        public System.Collections.Generic.IDictionary<string, float> ProviderIntervals { get; set; } = new Dictionary<string, float>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Let's nest the intervals under an object to allow for future default properties e.g.:

```json
"GlobalCounter": {
  "Providers": {
    "System.Runtime": {
      "Interval": 10
    }
  }
}
```